modules/vcflib/vcfbreakmulti/meta.yml

```diff
@@ -0,0 +1,45 @@
+name: "vcflib_vcfbreakmulti"
+description: write your description here
```

Comment:
Can you put a description of the `vcfbreakmulti` tool here?

Reply:
Done